pkg/repo/chartrepo.go

```diff
@@ -260,3 +263,8 @@ func ResolveReferenceURL(baseURL, refURL string) (string, error) {
 	parsedBaseURL.Path = strings.TrimSuffix(parsedBaseURL.Path, "/") + "/"
 	return parsedBaseURL.ResolveReference(parsedRefURL).String(), nil
 }
+
+func (e *Entry) String() string {
```

Comment:
is this needed? I don't see a reference to this anywhere.

Reply:
Yes, it seems so - it seems like the struct now including a bool cannot be converted to a string here: https://github.com/xvzf/helm/blob/393fd5fe173540703df320e267b8d706aa9e61b9/cmd/helm/repo_add_test.go#L144